### PR TITLE
Adding link to website repository in the footer

### LINKED ISSUE #49: Add link pointing to simple-icons-website repository
### Kind of issue

- [x] Feature
- [ ] Bug
- [ ] Design
- [ ] Other, namely:

### Description
I miss a link that takes you to the web repository (this repository), as long as it does not confuse with the main repository. Do you see it too meta? Maybe it could be "hidden" at the end, like the Twitter share button ... :thinking:

### PR DESCRIPTION
**Pull request by jorgeamadosoria**
Added a link to the website repository on github:

image

image

### CODE REVIEW

**Review by ericcornelissen — COMMENTED** [review]
Should we add an end-to-end test for this as well?

**Comment by jorgeamadosoria** [comment]
> Should we add an end-to-end test for this as well?

not sure. If you think it warrants it, sure.

IMO, it's just a link, there is little to test automatically. But it can be done if you want, I'm not against it per se.

**Comment by jorgeamadosoria** [comment]
I added a test case in the same test as the menu, since that test is for "external links".

File: public/index.pug [hunk 1]
```diff
@@ -175,5 +175,7 @@ html(lang="en-US")
           title Twitter logo
           path(d=twitterIcon.path)
         span Share this on Twitter!
-
+      div(class="footer-repo")
+          p. 
+            #[a(title="github repository" href="https://github.com/simple-icons/simple-icons-website") Made with ❤️ on GitHub]
```
Comment (ericcornelissen) [inline public/index.pug]: Is `❤️` a [UTF-8](https://github.com/simple-icons/simple-icons-website/blob/6e919f786dd4413327113b2099a88524c7c99e96/public/index.pug#L4) character?

File: public/stylesheet.css [hunk 2]
```diff
@@ -748,6 +748,7 @@ body.order-by-relevance .grid-item {
   .footer .footer-share {
     flex-grow: 1;
     justify-content: center;
+    margin-bottom: 0.3rem;
```
Comment (ericcornelissen) [inline public/stylesheet.css]: I feel like these are still a bit close together on smaller screens: 

image

Maybe we can increase the value a bit here, e.g.

```suggestion
    margin-bottom: 0.8rem;
```

File: public/index.pug [hunk 3]
```diff
@@ -175,5 +175,7 @@ html(lang="en-US")
           title Twitter logo
           path(d=twitterIcon.path)
         span Share this on Twitter!
-
+      div(class="footer-repo")
+          p.
```
Comment (ericcornelissen) [inline public/index.pug]: Minor but could you remove the trailing space here before someone else's editor (e.g. mine) removes it? (I really thought Prettier would take care of this :unamused:)

```suggestion
          p.
```

Comment (jorgeamadosoria) [reply public/index.pug]: yes, I don't know what happened there.

File: public/index.pug (hunk 1 above)
Comment (jorgeamadosoria) [reply public/index.pug]: It's a HTML entity now.

File: tests/e2e.test.js [hunk 4]
```diff
@@ -110,6 +110,12 @@ describe('External links', () => {
       await expect(page).toClick('a', { title });
     }),
   );
+
+  const footerRepositoryTitle = 'github repository';
+
+  it(`is possible to click the link for Github repository`, async () => {
+    await expect(page).toClick('a', { footerRepositoryTitle });
```
Comment (ericcornelissen) [inline tests/e2e.test.js]: This test doesn't seem to work, changing `footerRepositoryTitle` doesn't affect it on my end. Currently, it seems to just tests if there is _any_ link that can be clicked. (and, actually, the same goes for line 110).

You can find the documentation for [`toClick` here](https://github.com/smooth-code/jest-puppeteer/blob/master/packages/expect-puppeteer/README.md#expectinstancetoclickselector-options). I assumed, `{ title }` would look at the `title` attribute, but it seems that's not a thing that's supported.

While we're talking about this, it's not quite clear to me why you define `footerRepositoryTitle` outside the scope of the test and why you use backticks for the test name.

In this case you can fix the test using, e.g.:

```suggestion
    await expect(page).toClick('a', { text: /Made with . on GitHub/ });
```

**Review by ericcornelissen — CHANGES_REQUESTED** [review]

File: public/stylesheet.css (hunk 2 above)
Comment (ericcornelissen) [reply public/stylesheet.css]: Hmm :thinking: seeing a similar problem at a slightly larger size (this is exactly 682px wide):

image

File: tests/e2e.test.js (hunk 4 above)
Comment (jorgeamadosoria) [reply tests/e2e.test.js]: fixed.

File: public/stylesheet.css (hunk 2 above)
Comment (ericcornelissen) [reply public/stylesheet.css]: @jorgeamadosoria, did you see this comment?

Comment (jorgeamadosoria) [reply public/stylesheet.css]: no. I just got notified now.

Comment (jorgeamadosoria) [reply public/stylesheet.css]: fixed

File: public/index.pug [hunk 5]
```diff
@@ -175,5 +174,7 @@ html(lang="en-US")
...
+            #[a(title="github repository" href="https://github.com/simple-icons/simple-icons-website") Made with &#10084; on GitHub]
```
Comment (ericcornelissen) [inline public/index.pug]: We should probably distinguish this `title` from [this one](https://github.com/simple-icons/simple-icons-website/blob/7843220e0a2b825dffc2e3f58fb9d41fae7e0e5b/public/index.pug#L32) more clearly, here's some suggestions:

- header: "project repository", "main repository", "icon repository"
- footer: "website repository", "website source code", "website source"

**Review by ericcornelissen — COMMENTED** [review]
This looks pretty good, got one minor thing left that we should probably take care of before merging this

File: public/index.pug (hunk 5 above)
Comment (jorgeamadosoria) [reply public/index.pug]: true. Done.

**Review by ericcornelissen — APPROVED** [review]
Cool, thanks for this @jorgeamadosoria :+1: 

Also, great to have those e2e tests fixed :tada:

**Pull request merged**